index.js

```diff
@@ -433,6 +435,19 @@ Speech.prototype.sub = function (alias, word) {
 };
 
 /**
+ * This method lets the user provide a saying that won't have ssml tags escaped.  Use this if you want to pass an ssml string directly 
+ * @param saying
+ * If quotes are needed in the string, use double quotes within single quotes or template literal strings
+ * @returns {Speech}
+ */
+Speech.prototype.sayWithSSML = function (saying) {
+    this._present(saying, "The saying provided to Speech#sayWithTags(..) was null or undefined.");
+    this._elements.push(this._escape(saying, true));
```

Comment:
Instead of calling this_escape, just append the saying that’s provided and also, the error message still says sayWithTags vs. sayWithSSML

Reply:
updated!